Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -35,10 +35,10 @@ def linkage
     args = linkage_args.parse
 
     CacheStoreDatabase.use(:linkage) do |db|
-      kegs = if args.named.to_kegs.empty?
+      kegs = if args.named.to_keg.empty?
         Formula.installed.map(&:any_installed_keg).reject(&:nil?)
       else
-        args.named.to_kegs
+        args.named.to_keg
```

Comment:
Didn't we get rid of the `to_keg` method? it should probably stay as `to_kegs`, right?

Reply:
This should be `to_default_kegs`. Accidentally missed this when I renamed `to_keg` to `to_default_kegs`. Thanks for the catch!